src/tests/test_pyodide.py

```diff
@@ -146,6 +146,11 @@ def eval_code(code, ns):
     )
     assert selenium.run("x = 99; 5") == [3, 5]
     assert selenium.run("7") == [99, 7]
+    selenium.run(
+        """
+        pyodide.eval_code = old_eval_code
+        """
+    )
```

Comment:
Ideally we would want this in a finally block, so it's executed even if the test fails.

I think it didn't affect others tests too much because it's toward the end of the test suite, in the order it's usually run.

Reply:
Right, good point. (We could also just remove this test entirely, I can't imagine why there would be a regression in this area.)